go/test/endtoend/cluster/vtctldclient_process.go

```diff
@@ -72,7 +73,9 @@ func (vtctldclient *VtctldClientProcess) ExecuteCommandWithOutput(args ...string
 			filterDoubleDashArgs(pArgs, vtctldclient.VtctldClientMajorVersion)...,
 		)
 		msg := binlogplayer.LimitString(strings.Join(tmpProcess.Args, " "), 256) // limit log line length
-		log.Infof("Executing vtctldclient with command: %v (attempt %d of %d)", msg, i+1, retries)
+		if !vtctldclient.Quiet {
```

Comment:
Where is this set? Or is it for future use?

Reply:
It is for future use. I have this in an ongoing change, and thought it will be useful to commit separately upstream.